### PR TITLE
ONVIF compatibility fix - use GetCapabilities instead of GetServices

### PR DESCRIPTION
**Pull request by ethan-beauc**
The IncludeCapability flag within GetServices was causing Faults on some cameras when included, and on others when excluded. Switching to GetCapabilities to retrieve the service addresses worked on all tested devices.

### CODE REVIEW

File: src/us/mn/state/dot/tms/server/comm/onvifptz/Service.java [hunk 1]
```diff
@@ -53,7 +54,7 @@ public abstract class Service {
 
 	/** Logger method */
 	protected void log(String s) {
-		OnvifPTZPoller.slog("PTZCommandProp:" + s);
```
Comment (DougLau) [inline src/us/mn/state/dot/tms/server/comm/onvifptz/Service.java:56]: This should be left alone -- in production, the onvif log is more appropriate than stdout.

File: src/us/mn/state/dot/tms/server/comm/onvifptz/DeviceService.java [hunk 2]
```diff
@@ -55,41 +55,33 @@ public String getServices() throws IOException {
 	}
 
 	/** Gets a service address by its namespace */
```
Comment (DougLau) [inline src/us/mn/state/dot/tms/server/comm/onvifptz/DeviceService.java:57]: Does it make sense to update this comment to "name"?

File: src/us/mn/state/dot/tms/server/comm/onvifptz/Service.java [hunk 3]
```diff
@@ -197,17 +198,15 @@ public String sendRequestDocument(Document doc) throws IOException {
...
-				StringBuilder response = new StringBuilder();
-				while ((inputLine = in.readLine()) != null) {
-					response.append(inputLine);
-				}
-				// Process the response here (XML parsing, etc.)
-				resp = response.toString();
-			}
+			BufferedReader in = new BufferedReader(new InputStreamReader(connection.getInputStream()));
+			resp = in.lines().collect(Collectors.joining("\n"));
+			System.out.println("Request succeeded, code: " + responseCode + " Response:\n" + resp);
```
Comment (DougLau) [inline src/us/mn/state/dot/tms/server/comm/onvifptz/Service.java:203]: This should also use the onvifptz logger

**Review by DougLau — CHANGES_REQUESTED** [review]

File: src/us/mn/state/dot/tms/server/comm/onvifptz/Service.java (hunk 1 above)
Comment (ethan-beauc) [reply src/us/mn/state/dot/tms/server/comm/onvifptz/Service.java]: Fixed - stdout is used in my standalone test application, forgot to replace those in production.

**Review by DougLau — APPROVED** [review]

**Pull request merged**